src/SFA.DAS.Commitments.Api.Types/DataLock/Types/DataLockErrorCode.cs

```diff
@@ -1,20 +1,32 @@
 ﻿using System;
+using System.ComponentModel;
```

Comment:
unused using - can be removed

Reply:
Comment obsoleted by actually including the DescriptionAttributes.